nilearn/image/resampling.py

```diff
@@ -346,6 +346,15 @@ def resample_img(img, target_affine=None, target_shape=None,
     **NaNs and infinite values**
     This function handles gracefully NaNs and infinite values in the input
     data, however they make the execution of the function much slower.
+
+    **Handling non-native endian in given Nifti images**
+    This function automatically changes the byte-ordering information
+    in the image dtype to new byte order. From non-native to native, which
+    implies that if the given image has non-native endianess then the output
+    data in nifti image will have native dtype. This is only valid when
```

Comment:
I think that, instead of "this is only valid", you mean "this is only the case".